README.md

```diff
@@ -6,6 +6,39 @@ Checks the official JSON files every once in a while (configurable in the config
 file) for changes and sends new warnings to registered users based on the
 coordinates the entered.
 
+## Install
+
+(tested on Linux Mint 20.2)
+
+1. Clone the repository
+
+`git clone https://github.com/jplitza/nina_xmpp.git`
+
+2. Install dependencies
+
+```
+sudo pip3 install aioxmpp httpx shapely geoalchemy2 argparse_logging
```

Comment:
This will already be handled by running `setup.py build`, won’t it?

Reply:
> This will already be handled by running `setup.py build`, won’t it?

Might be, the dependencies are listed in setup.cfg as far as i can see, but it didn't work for somehow... I'm no python expert though.